File: p_c/send.py
import pika
while(1):
  connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
  channel = connection.channel()

  channel.queue_declare(queue='666')
  message = raw_input("please enter what you want: ")
  channel.basic_publish(exchange='',
                        routing_key='666',
                        body = message)
  print(" [x] Sent 'Hello World!'")
  connection.close()

File: helloworld/send.cpp
#include <SimpleAmqpClient/SimpleAmqpClient.h>
#include <iostream>
#include <string>
using namespace std;
int main() {
  string queue = "hi";
  AmqpClient::Channel::ptr_t channel = AmqpClient::Channel::Create();
  channel->DeclareQueue(queue, false, true, false, false);
  string str = "hi!";
 channel->BasicPublish("", queue, AmqpClient::BasicMessage::Create(str));
  cout << "sent hello world";
}